tests/fdb/test_fdb.py

```diff
@@ -150,25 +151,46 @@ def setup_fdb(ptfadapter, vlan_table, router_mac, pkt_type):
     time.sleep(FDB_POPULATE_SLEEP_TIMEOUT)
     # Flush dataplane
     ptfadapter.dataplane.flush()
+
     return fdb
 
 
-@pytest.fixture
+def get_fdb_dynamic_mac_count(duthost):
+    res = duthost.command('show mac')
+    logger.info('"show mac" output on DUT:\n{}'.format(pprint.pformat(res['stdout_lines'])))
+    total_mac_count = 0
+    for l in res['stdout_lines']:
```

Comment:
Really minor nitpick, but I would use double-quotes here to be consistent w/ the rest of the file. I do think using single quotes in the log statement to avoid having to add escape characters for the quotes is OK though.

Reply:
@daall As discussed offline we will not do the cleanup this time to ease the fix review for this PR.  Perhaps a generic PR to go over all test cases to clean up the format with syntax consistency issues.